apps/server/src/modules/board/controller/dto/element/create-content-element.body.ts

```diff
@@ -9,6 +9,7 @@ export class CreateContentElementBody {
 		enum: ContentElementType,
 		required: true,
 		nullable: false,
+		enumName: 'ContentElementType',
```

Comment:
Für mich ist unklar warum sowohl das enum als auch enumName mitgegeben werden muss. 
Wenn das wirklich notwendig ist, wäre evtl. ein kleiner Kommentar mit dem Grund dahinter hilfreich.

Reply:
https://docs.nestjs.com/openapi/types-and-parameters#enums-schema
Wenn das um ein der selbe Enum geht, kann man mit enumName Enum Name definieren um folgendes zu vermeiden:

TextElementContentBodyTypeEnum und FileElementContentBodyTypeEnum sind von selben Type ContentElementType